src/formatting/unused_parents.rs

```diff
@@ -434,7 +575,6 @@ y = 2
 [parent.child]
 ",
             str![[r#"
-[parent]
 # body comment
```

Comment:
Isn't a comment like this part of `parent`, just like a comment on the same line?